chapter02/worksheets/solution2.1.7.4_AR.scala

```diff
@@ -0,0 +1,21 @@
+/** author: Andreas Röhler */
+
+/** 
+  Exercise 2.1.7.4
+  Convert a Seq[Int] into a Seq[(Int, Boolean)] where
+  the Boolean value is true when the element is followed by a larger
+  value. For example, the input sequence Seq(1,3,2,4) is to be
+  converted into Seq((1,true),(3,false),(2,true),(4,false)). (The last
+  element, 4, has no following element.)
+ */
+
+val a = Seq[Int](1, 3, 2, 4)
+val b = a.last
+val c: Seq[(Int, Boolean)] = Seq((4, false))
```

Comment:
I would expect that `c` uses `b` rather than just hard-code `4`.

Also, it would help if you rewrite this as a function.

`def followedByGreater(a: Seq[Int]): Seq[(Int, Boolean)] = ...`